io.github.harmonoid.yml

```diff
@@ -0,0 +1,88 @@
+app-id: io.github.harmonoid
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Platform
+rename-icon: harmonoid
+rename-appdata-file: harmonoid.appdata.xml
+rename-desktop-file: harmonoid.desktop
+command: harmonoid
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-music
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: libmediainfo
+    subdir: Project/GNU/Library
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --with-libcurl
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: https://github.com/MediaArea/MediaInfoLib/archive/refs/tags/v21.03.tar.gz
+        sha256: b083c113adb80f6608d5e86d1be07f9abd4257bd0e8456c8f9eb8756a385c5a0
+
+  - name: webkitgtk
+    buildsystem: cmake-ninja
+    build-options:
+      arch:
+        x86_64:
+          cflags: -g1
+          cxxflags: -g1
+          config-opts:
+            - -DCMAKE_BUILD_TYPE=Release
+        aarch64:
+          config-opts:
+            - -DCMAKE_BUILD_TYPE=MinSizeRel
+    config-opts:
+      - -DPORT=GTK
+      - -DUSE_LIBNOTIFY=OFF
+      - -DUSE_LIBSECRET=OFF
+      - -DUSE_WOFF2=OFF
+      - -DUSE_WPE_RENDERER=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DENABLE_SPELLCHECK=OFF
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.32.3.tar.xz
+        sha256: c1f496f5ac654efe4cef62fbd4f2fbeeef265a07c5e7419e5d2900bfeea52cbc
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+
+  - name: harmonoid
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 harmonoid/harmonoid /app/harmonoid/harmonoid
+      - chmod +x /app/harmonoid/harmonoid
+      - cp -R harmonoid/usr/share /app/share
+      - cp -R harmonoid/lib /app/harmonoid/lib
+      - cp -R harmonoid/data /app/harmonoid/data
+      - sed -i 's/<id>harmonoid/<id>io.github.harmonoid/' /app/share/metainfo/harmonoid.appdata.xml
+      - mkdir -p /app/bin/ /app/lib/
+      - ln -s /app/harmonoid/harmonoid /app/bin/harmonoid
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
```

Comment:
you should also disable the aarch64 build. But why not building from source instead?

Reply:
ah, flutter, again.

Reply:
@hfiguiere I am testing building app within flathub bot here https://github.com/flathub/flathub/pull/2509, but still CI fails can you help me debug that.